sorc/ncep_post.fd/AllGETHERV_GSD.f

```diff
@@ -45,12 +45,12 @@ SUBROUTINE AllGETHERV(GRID1)
            ibufsend(ij)=GRID1(i,j)
         enddo
      enddo
-     if(ij /= RECVCOUNTS(me+1)) then
+     if(ij .ne. RECVCOUNTS(me+1)) then
```

Comment:
@junwang-noaa We have unified relational operators as (>, <, /=, ==) from the UPP refactor. I would like to know if there is a specific reason of using ".ne." instead of "/=" here?

Reply:
I was using an old code, the original code is restored.